Finalise UI for About/Contribute/Teachers Page

aayush-05 opened the pull request:
### Describe the changes you have made in this PR -
Made About/Contribute page fully responsive across all screen sizes and also refactored their code.

### Video Recording -
https://drive.google.com/file/d/1yHszGq66QyrffuoNSxxDdnKv56KO3jnd/view

Teachers Page:
![CircuitVerse - Teachers](https://user-images.githubusercontent.com/47032027/82738651-d97fc880-9d56-11ea-8bf9-a38b2fe58927.gif)

aayush-05 commented:
@nitin10s @tachyons have a look at this; let me know if any screenshots required. I have added a video recording because showing those many screen sizes for each part would end up in a lot of screenshots.
Also @tachyons can I ignore the codeclimate issues for oldStyles.scss, I have made this file as a compilation of extra/irrelevant styles and it will simply be discarded when all the code would be refactored ?

aayush-05 commented on app/assets/stylesheets/layout.scss:
```diff
@@ -143,3 +139,42 @@ body {
+}
+
+.paragraph-text-justified {
+  text-align: justify;
```
Done

aayush-05 commented on app/assets/stylesheets/layout.scss:
```diff
@@ -143,3 +139,42 @@ body {
+}
+
+.row-text {
+  text-align: center;
```
Done

nitin10s commented on app/views/about/index.html.erb:
```diff
@@ -1,60 +1,74 @@
+  <br>
+  <div class="row row-text">
+    <div class="col-xs col-sm col-md col-lg text-center">
+      <a class="btn button-primary about-button-contribute" href="https://opencollective.com/CircuitVerse" target="_blank">Become a Contributor</a>
```
This button would redirect user to the Contribute Page.

aayush-05 replied:
Done

aayush-05 commented:
@tachyons I have fixed all the codeclimate issues; but idk why the coverage is decreased

tachyons commented:
Not a blocker for merging

aayush-05 commented:
Okay, then please review this one and merge

aayush-05 commented:
@tachyons @nitin10s I had added refactored Teachers Page in this PR as well. Please review and let me know if any changes required.
(@nitin10s I have added shadow to the images on Teachers Page rather than a border, let me know if its fine)

nitin10s commented:
1. On clicking the images, do they expand?
2. The sub-heading "CircuitVerse has been designed to be very easy to use in class. The platform has features to assist teachers in class and assignments." looks bolder than it is currently, please make the appropriate changes.

aayush-05 commented:
1. No, the size of image doesn't change. On hover, the amount of shadow doubles.
2. I haven't changed the font-weight or size for that. May be it is looking like that in the gif; refer this
![image](https://user-images.githubusercontent.com/47032027/82744493-ed4b1f00-9d96-11ea-9c55-16b6126edb1a.png)

nitin10s commented:
Shadows are provided when we want to hint user that the item is clickable. I agree this looks better than the outlines but it gives our wrong cues.

aayush-05 commented:
Ah, okay. Shall I revert them back to borders then ?

nitin10s commented:
Yeah

aayush-05 commented:
Done @nitin10s 
![image](https://user-images.githubusercontent.com/47032027/82786345-0ed50500-9e82-11ea-9139-e27aa37d4c8e.png)

tachyons commented on app/assets/stylesheets/logixRelease.scss:
```diff
@@ -1,3 +1,7 @@
+body {
+  padding-top: 90px;
```
Curious Why ?

aayush-05 replied:
So that the top content doesn't overlap with the navbar

tachyons commented on app/assets/stylesheets/utilities.scss:
```diff
@@ -0,0 +1,51 @@
+//buttons
+.button-primary {
```
Are you overriding bootstrap class here ?

aayush-05 replied:
No, the bootstrap one is btn-primary. It is the class I made for our custom buttons; so that then it is just adding this class for all other pages without having additional styles

tachyons replied:
may be `components.scss` ?

aayush-05 replied:
Done
